Documentation/networking.md

```diff
@@ -21,9 +21,13 @@ rkt will allocate an IPv4 /31 (2 IP addresses) out of 172.16.28.0/24 and assign
 It will additionally set the default route in the pod namespace.
 Finally, it will enable IP masquerading on the host to NAT the egress traffic.
 
-**Note**: The default network must be explicitly listed in order to be loaded when `-private-net=...` is specified with a list of network names. 
-In case it's not specified, a restricted default network will be created to allow communication with the metadata service. 
-In this case the default route and IP masquerading will not be setup.
+**Note**: The default network must be explicitly listed in order to be loaded when `--private-net=...` is specified with a list of network names.
+
+Example: If you want default networking and two more networks you need to pass `--private-net=default,net1,net2`.
+
+In case *default* is not among the specified networks the *default-restricted* network will be added to the list of networks automatically.
```

Comment:
you can add that: *default-restricted* is also used with no nets passed at all eg. ```rkt run --private-net --local```

(this case confuses me over and over again :smiley:  )